app/models/planting.rb

```diff
@@ -112,6 +112,21 @@ def planted?(current_date = Date.current)
     planted_at.present? && current_date.to_date >= planted_at
   end
 
+  def finished_at_date_passed?
+    finished_at.present? && (finished_at - Date.current).to_i <= 0
+  end
+
+  def days_until_finished
+    return 0 if finished?
```

Comment:
No, it's a boolean column on that table of the database. 
```
    t.boolean  "finished",             default: false
```
So, this is checking, is that set to true -- which seems to be yet another way a planting record can be finished.